python/script.py

```diff
@@ -0,0 +1,11 @@
+import pytest
```

Comment:
I assume this file is for temporary testing and will be deleted.

Reply:
Yes